packages/mobile/src/firebase/saga.ts

```diff
@@ -113,7 +111,7 @@ export function* subscribeToCeloGoldExchangeRateHistory() {
   yield call(waitForFirebaseAuth)
   const history = yield select(exchangeHistorySelector)
   const latestExchangeRate = history.celoGoldExchangeRates[history.celoGoldExchangeRates.length - 1]
-  const chan = yield call(celoGoldExchangeRateHistoryChannel, latestExchangeRate)
+  const chan = yield call(celoGoldExchangeRateHistoryChannel, history.lastTimeUpdated)
```

Comment:
Does `lastTimeUpdated` always equal `lastExchangeRate.timestamp`? If so, we do we store both?

Reply:
`lastExchangeRate` is just the last element of the array of `exchangeRate` we are storing, so calling `.timestamp` on it returns the last known timestamp we have saved. 
`history.lastTimeUpdated` returns the time we received the data from Firebase and updated the store last.

In practice, it shouldn't make a difference, but I just wanted to change something because the previous implementation was failing sometimes and causing the whole history to be stored several times.

Now that you mention this though, we're using `Date.now()` which depends on the local clock afaik, I should probably change that to `getRemoteTime()`

Reply:
Interesting, it looks like we can even go as far as not persisting the exchange rate history in Redux, as I think it's already cached at the firebase lib layer. We should open an issue with this.